### ISSUE TITLE
Perl (and other languages plugins) broken in High Sierra / 10.13

### ISSUE THREAD

**Issue by agneau** [xchataqua/xchataqua#257]
After upgrading from mac os x 10.9.5 (very old, I know) to the latest 10.13.2. The perl plugin stop to works. 
I remove the old xchat azure and install the latest xchat azure 1.18.11 from appstore.
When I try to load the perl plugin, it fails with 

dlopen(/Applications/XChat Azure.app/Contents/PlugIns/perl.bundle/Contents/MacOS/perl, 10): Symbol not found: _xchat_command
     Referenced from: /Applications/XChat Azure.app/Contents/PlugIns/perl.bundle/Content/MacOS/perl
 Expected in: flat namespace
    in /Applications/XChat Azure.app/Contents/PlugIns/perl.bundle/Contents/MacOS/perl

/Applications/XChat\ Azure.app/Contents/PlugIns/perl.bundle/Contents/Info.plist contains:
	<key>DTSDKName</key>
	<string>macosx10.12</string>

It looks like a new version need to be build for 10.13 .

**Is there a new build for 10.13 with working perl plugin somewhere (not necessary in appstore) ?**

Il don(t use other languages plugins, but it seems python,ruby,tcl must have the same problem:
dlopen(/Applications/XChat Azure.app/Contents/PlugIns/python.bundle/Contents/MacOS/python, 10): Symbol not found: _g_build_filename
     Referenced from: /Applications/XChat Azure.app/Contents/PlugIns/python.bundle/Contents/MacOS/python
     Expected in: flat namespace
    in /Applications/XChat Azure.app/Contents/PlugIns/python.bundle/Contents/MacOS/python

dlopen(/Applications/XChat Azure.app/Contents/PlugIns/ruby.bundle/Contents/MacOS/ruby, 10): Library not loaded: /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/libruby.2.0.0.dylib
     Referenced from: /Applications/XChat Azure.app/Contents/PlugIns/ruby.bundle/Contents/MacOS/ruby

dlopen(/Applications/XChat Azure.app/Contents/PlugIns/tcl.bundle/Contents/MacOS/tcl, 10): Symbol not found: _xchat_command
     Referenced from: /Applications/XChat Azure.app/Contents/PlugIns/tcl.bundle/Contents/MacOS/tcl
  Expected in: flat namespace

**Comment by Mirppc** [comment]
Nah, i am having a spike nope issue with Xchat Azure and Perl.  Didnt notice this otherwise i would have posted here.  Effects Sierra as well.
